<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>231</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>One</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Two</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>2</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>The</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>Sky</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>is</string>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string>Blue</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="list_items_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>401</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>List items</string>
   </property>
  </widget>
  <widget class="QPushButton" name="set_item_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>401</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Set item</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
